src/rules/noConsecutiveBlankLinesRule.ts

```diff
@@ -43,6 +67,9 @@ class NoConsecutiveBlankLinesWalker extends Lint.SkippableTokenAwareRuleWalker {
     public visitSourceFile(node: ts.SourceFile) {
         super.visitSourceFile(node);
 
+        const options = this.getOptions();
+        const allowedBlanks = options && options[0] || Rule.DEFAULT_ALLOWED_BLANKS;
```

Comment:
The `options` property of the  `NoConsecutiveBlankLinesWalker` class _is_ an array, and it is the rule's arguments. In it's parent's parent:
```typescript
export class RuleWalker extends SyntaxWalker {
    ...
    private options: any[];
    ...

    constructor(private sourceFile: ts.SourceFile, options: IOptions) {
        ...
        this.options = options.ruleArguments;
        ...
    }

    ...
}
```